src/batou/deploy.py

```diff
@@ -121,6 +121,8 @@ def __init__(
         self.predict_only = predict_only
         self.jobs = jobs
 
+        self.timer = Timer("deployment")
```

Comment:
Maybe unifying this with the component timer would be helpful, too, so we can see what time we needed to connect/configure and actually deploy.

This would mean that a Timer has a total and an arbitrary set of steps (which means the earlier comment that uses a hard-coded step in an attribute (like `human_total` or `human_verify`) should rather be a callable like `timer.humanize("verify")`.

And maybe it can even have a formatter method that if you don't pass anything you get:

```
>>> timer.humanize()
total=..., step1=..., step2, ... 
```

The order of steps could rely on dict order nowadays.